### PR TITLE
Cthulhu Pack Iteration for more clear in-pack theme and synergy

### PR DESCRIPTION
**Pull request by mikemayhemdev**
End of Days reworked, now generates dark based on unspent energy.

Gaze the Void +1 base damage. Nightmare Stance reworked - when exited, draw 2 cards, gain E, and lose 1 HP. Exits automatically when you use Madness.

Familial Curse changed - Instead of discarding the hand, grants Madness and 1 HP loss at the start of next turn.

Star Spawn - Increase max orb traded for a Heal effect. Costs 3 (up from 2).

### CODE REVIEW

File: src/main/java/thePackmaster/powers/cthulhupack/EndOfDaysPower.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    public EndOfDaysPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+        updateDescription();
+    }
+
+    @Override
+    public void atEndOfTurn(boolean isPlayer) {
+        if (isPlayer) {
+            if (EnergyPanel.getCurrentEnergy() > 1) flash();
```
Comment (erasels) [inline src/main/java/thePackmaster/powers/cthulhupack/EndOfDaysPower.java:28]:
```suggestion
            if (EnergyPanel.getCurrentEnergy() > 0) flash();
```

File: src/main/resources/anniv5Resources/localization/eng/cthulhupack/Stancestrings.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "${ModID}:Nightmare": {
     "NAME": "Nightmare",
     "DESCRIPTION": [
-      "Lose 1 HP and draw a card at the start of each turn. When you exit Nightmare Stance, add a Madness to your hand."
+      "When you use 'Madness', exit Nightmare Stance. NL When you exit Nightmare Stance, draw 2 cards gain [E], and lose 1 HP."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/cthulhupack/Stancestrings.json:5]:
```suggestion
      "When you play #yMadness, exit #yNightmare #yStance. NL When you exit #yNightmare #yStance, draw #b2 cards gain [E] , and lose #b1 HP."
```

File: src/main/resources/anniv5Resources/localization/eng/cthulhupack/Powerstrings.json [hunk 3]
```diff
@@ -20,5 +20,12 @@
       " and take ",
       " damage."
     ]
+  },
+  "${ModID}:EndOfDaysPower": {
+    "NAME": "End of Days",
+    "DESCRIPTIONS": [
+      "When you end your turn with energy, spend it all. #yChannel #b",
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/cthulhupack/Powerstrings.json:27]: This is wrong. The energy is not being spent.

Comment (erasels) [reply src/main/resources/anniv5Resources/localization/eng/cthulhupack/Powerstrings.json:27]: Also text highlighting please

File: src/main/resources/anniv5Resources/localization/eng/cthulhupack/Keywordstrings.json [hunk 4]
```diff
@@ -4,6 +4,6 @@
     "NAMES": [
       "nightmare"
     ],
-    "DESCRIPTION": "Stance: Lose 1 HP and draw a card at the start of each turn. NL When you exit this stance, add #yMadness to your hand."
+    "DESCRIPTION": "Stance: When you use 'Madness', exit Nightmare Stance. NL When you exit Nightmare Stance, draw 2 cards, gain [E] , and lose 1 HP."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/cthulhupack/Keywordstrings.json:7]: text highlighting

**Review by erasels — CHANGES_REQUESTED** [review]

**Comment by modargo** [comment]
Changes look good, merging this in.

**Pull request merged**